Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftWindowsEventFirewall.yaml

```diff
@@ -96,56 +99,68 @@ ParserQuery: |
     , endtime:datetime=datetime(null)
     , srcipaddr_has_any_prefix:dynamic=dynamic([])
     , dstipaddr_has_any_prefix:dynamic=dynamic([])
+    , ipaddr_has_any_prefix:dynamic=dynamic([])
     , dstportnumber:int=int(null)
     , hostname_has_any:dynamic=dynamic([])
     , dvcaction:dynamic=dynamic([])
     , eventresult:string='*'
     , disabled:bool=false
       )
-      { 
+      {    
+      let src_or_any=set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix); 
```

Comment:
Just noticed: any way to materialize the unions? I recall materialize works only on tables :-( correct?

Reply:
- you are right. I thought of that too. materializing constants (scalars) is needed. I could:
let x=materialize( **print** (set_union(s1, s2) ) )

not sure at all that this is preferred